src/commands/runDraftTool/helper/languages.ts

```diff
@@ -0,0 +1,41 @@
+export interface DraftLanguage {
+   name: string;
+   id: string;
+   versions: string[];
+}
+export const draftLanguages: DraftLanguage[] = [
```

Comment:
Note that this is temporary. This will ideally be read from a draft info command